docs/overview/schema/predicates.mdx

```diff
@@ -35,6 +35,7 @@ Below are the built-in predicates for the `_predicate` collection. As mentioned
 | `_predicate/restrictTag`        | `boolean` | (optional) Only applicable to predicates of type `tag`. If true, a tag, which corresponds to this predicate object must exist before adding predicate-object pair.                                                                                                                                                                                                                                                                                                                                      |
 | `_predicate/encrypted`          | `boolean` | (Not in production yet, optional) Expects the value to come in as an encrypted string. Type checking will be disabled, and ledger functions won't be permitted on this value.                                                                                                                                                                                                                                                                                                                           |
 | `_predicate/fullText`           | `boolean` | (optional) If true, full text search enabled on this ledger. By default, the language for a Fluree ledger is set to English. You can change the default language in [ledger settings](/overview/schema/settings.md).                                                                                                                                                                                                                                                                                    |
+| `_predicate/retractDuplicates`           | `boolean` | (optional) If false (default), when a transaction creates a new flake that already exists it does not update the existing data. When true, it will always force a retraction/insertion.                                                                                                                                                                                                                                                                                    |
```

Comment:
Tiny nitpick, but can you scoot the `|` after the pred name back to the left to line up with the ones above it?